src/main/scala/com/gu/datalakealerts/Features.scala

```diff
@@ -7,9 +7,10 @@ import com.gu.datalakealerts.Platforms.{ Android, iOS, Platform }
 
 object Features {
 
-  val yesterday: LocalDate = LocalDate.now().minusDays(1)
   val allFeaturesWithMonitoring: List[Feature] = List(FrictionScreen, OlgilEpic, BrazeEpic)
 
+  val yesterday: LocalDate = LocalDate.now().minusDays(1)
```

Comment:
shouldn't this be a def?

Objects in scala are singletons, loaded only once in memory. So if a lambda happen to be kept alive by AWS for more than 1 day, the code will be processing the wrong data.

Reply:
Good point - thanks 👍